l10n/en/products/vpn/platforms/shared.ftl

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/desktop/
+
+# Shared
+vpn-shared-title = { -brand-name-mozilla-vpn }
```

Comment:
We should use a consistent name space for the strings in this Fluent file, so it's more obvious where they come from. Suggest using `vpn-platforms-shared-`?

We can consolidate this into a real shared file for all VPN pages once we're ready to localize.

Reply:
Good call. Updating!

Reply:
Revised everything into the `vpn-platforms-shared-` format!